src/lib/mina/fixtures/fetch-actions-response-with-transaction-info.ts

```diff
@@ -0,0 +1,1864 @@
+export { mockFetchActionsResponse };
```

Comment:
see what i wrote above; this could save us a few thousand lines here too

Reply:
FWIW, this is the block in which the actions are out of order by the currently-live algorithm.  So if we did trim it, we should at least keep this.

```
      {
        blockInfo: {
          distanceFromMaxBlockHeight: 10037,
        },
        actionState: {
          actionStateOne:
            '11553781833537412187998791170094246504179739906426596650283148421062134031747',
          actionStateTwo:
            '2657110063538905863496230550574434206907731812034404344231638673775478011856',
        },
        actionData: [
          {
            accountUpdateId: '40618',
            data: [
              '7',
              '7',
              '7',
              '7',
              '7',
              '7',
              '15661593045172372274349484602975176160403678593916071611247487060653084577990',
              '0',
              '1',
            ],
            transactionInfo: {
              sequenceNumber: 4,
              zkappAccountUpdateIds: [40618, 6396, 33066],
            },
          },
          {
            accountUpdateId: '40617',
            data: [
              '3',
              '4',
              '5',
              '7',
              '6',
              '2',
              '6453147199940115032546911372596674482366995380739961315278593325733504245402',
              '0',
              '1',
            ],
            transactionInfo: {
              sequenceNumber: 5,
              zkappAccountUpdateIds: [40617, 40372, 40373],
            },
          },
          {
            accountUpdateId: '40619',
            data: [
              '3',
              '4',
              '5',
              '7',
              '6',
              '3',
              '21594276700301685638451290563121567472719745179788000956082286147863718718533',
              '1',
              '1',
            ],
            transactionInfo: {
              sequenceNumber: 6,
              zkappAccountUpdateIds: [40619, 33539, 33540],
            },
          },
          {
            accountUpdateId: '40620',
            data: [
              '3',
              '4',
              '5',
              '7',
              '6',
              '3',
              '21594276700301685638451290563121567472719745179788000956082286147863718718533',
              '1',
              '1',
            ],
            transactionInfo: {
              sequenceNumber: 7,
              zkappAccountUpdateIds: [40620, 33539, 33540],
            },
          },
        ],
      },
      ```

Reply:
Sorry, I meant the property testing would be benefical here too